docs/en/metrics/infra-ui-intro.asciidoc

```diff
@@ -2,11 +2,11 @@
 [role="xpack"]
 == {metrics-app}
 
-After you have <<install-infrastructure-monitoring, started monitoring metrics>> and data is streaming to {es},
+After <<install-infrastructure-monitoring, setting up metrics monitoring>> and data is streaming to {es},
```

Comment:
The first part of this sentence sounds weird to me.

> After setting up metrics monitoring and data is streaming to {es}, you can use the {metrics-app} to view metrics from your infrastructure and identify problems in real-time.

I wonder if we can just ax the first half of the sentence and say something like:

> Use the {metrics-app} to view metrics from your infrastructure and identify problems in real-time.

Reply:
Great suggestion. Thanks.